cache/s3/s3.go

```diff
@@ -62,17 +64,19 @@ func New(config map[string]interface{}) (cache.Interface, error) {
 	//	parse the config
 	c := dict.M(config)
 
-	//	TODO: this could be cleaner
-	defaultMaxZoom := 0
+	// the config map's underlying value is int
+	defaultMaxZoom := tegola.MaxZ
 	maxZoom, err := c.Int(ConfigKeyMaxZoom, &defaultMaxZoom)
```

Comment:
We can change this to parse for a Uint and then drop the less than 0 check on line 74

Reply:
Ah, missed those